Dockerfile.apiserver

```diff
@@ -1,21 +1,27 @@
-# syntax=docker/dockerfile:1.2
+# syntax=docker/dockerfile:1
```

Comment:
> We recommend using `docker/dockerfile:1`, which always points to the latest stable release of the version 1 syntax, and receives both "minor" and "patch" updates for the version 1 release cycle. 

Taken from [here](https://docs.docker.com/build/dockerfile/frontend/).

@Tehsmash  ^

Reply:
as long as it will update it, when I originally looked at this if you specified 1 then it would happy use whatever 1.X you had locally instead of implying latest 1.X. Which meant if you had 1.1 locally it would be OK with it.